### PR TITLE
misc qa tickets

### PR DESCRIPTION
**Pull request by jaslakson**
misc ui fixes

### CODE REVIEW

**Review by ttornese — APPROVED** [review]

File: src/components/Cards/RelatedDocumentCard/index.js [hunk 1]
```diff
@@ -99,10 +108,15 @@ const RelatedDocumentCardSubtitleTheme = {
...
+    ${mixins.styledLink(color.turquoise, color.seaSalt)}
 
     ${breakpoint('md')`
-      display: block;
+      align-self: flex-start;
+      display: inline-block;
     `}`,
+  atk: css`
+    ${mixins.styledLink(color.turquoise, color.seaSalt)}
+  `,
```
Comment (mirdathebirda) [inline src/components/Cards/RelatedDocumentCard/index.js:119]: we can skip this declaration if it's already set in default

**Review by mirdathebirda — APPROVED** [review]
👍

File: src/components/Cards/RelatedDocumentCard/index.js (hunk 1 above)
Comment (jaslakson) [reply src/components/Cards/RelatedDocumentCard/index.js:119]: I think I meant to move it to atk and remove the default one...I'll do that now

**Comment by atk-dm** [comment]
:rocket: PR was released in [`v1.5.0`](https://github.com/Americastestkitchen/mise-ui/releases/tag/v1.5.0) :rocket:

**Pull request merged**